.github/workflows/tests.yml

```
@@ -19,5 +19,4 @@ jobs:
       - uses: actions/checkout@dcd71f646680f2efd8db4afa5ad64fdcba30e748
       - name: Run tests
         run: make test-exercises
-        # TODO: fix generators
-        #run: make test-generator
+        run: make test-generator
```

In that case, I think we had better just make it a separate Step (in the GitHub Actions sense of the word)

```suggestion
      - name: Run generator tests
        run: make test-generator
```

again, my apologies for making the original comment wrong

---

@marksiemers Could you apply peter's suggestion? That should fix things